tests/unit_test/mocks.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a MIT license that can be found in the LICENSE file.
 // SPDX-License-Identifier: MIT
 
-package deepgram_test
```

Comment:
uh oh... this will make the tests get published with the deepgram package. i don't think that is right

Reply:
Yeah I had some weird errors with `deepgram_test` so I was playing around and committed this incorrectly. Good catch. Fixed in recent commit.